### PR TITLE
LW-9423 proposal procedures details

### PR DESCRIPTION
**Pull request by iccicci**
# Context

**db-sync** v3.0.0 introduced JSON encoded governance action proposals

# Proposed Solution

Parsed and translated the governance action proposals from db-sync to our core types.

### CODE REVIEW

File: packages/cardano-services/environments/.env.sanchonet [hunk 1]
```diff
@@ -1,4 +1,4 @@
 HANDLE_POLICY_IDS=${HANDLE_POLICY_IDS:-f0ff48bbb7bbe9d59a40f1ce90e9e9d0ff5002ec48f232b49ca0fb9a}
 OGMIOS_PORT=${OGMIOS_PORT:-1341}
 METADATA_FETCH_MODE="direct"
-SMASH_URL="http://cardano-smash:3100/api/v1"
```
Comment (mkazlauskas) [inline packages/cardano-services/environments/.env.sanchonet:4]: Is SMASH_URL not used anywhere?

**Review by mkazlauskas — APPROVED** [review]
Looks good! Just a question below

File: packages/cardano-services/environments/.env.sanchonet (hunk 1 above)
Comment (mchappell) [reply packages/cardano-services/environments/.env.sanchonet:4]: It was removed in #1056

File: packages/cardano-services/src/ChainHistory/DbSyncChainHistory/ChainHistoryBuilder.ts [hunk 2]
```diff
@@ -52,26 +52,99 @@ import {
...
+            coldCredential: credentialFromDbSync(Object.fromEntries([key.split('-')])),
+            epoch: value as Cardano.EpochNo
+          }))
+        ),
+        membersToBeRemoved: new Set((contents[1] as DbSyncCredential[]).map(credentialFromDbSync)),
+        newQuorumThreshold: {
+          denominator: Number.parseInt(denominator!, 10),
+          numerator: Number.parseInt(numerator!, 10)
+        }
+        // missing
```
Comment (mchappell) [inline packages/cardano-services/src/ChainHistory/DbSyncChainHistory/ChainHistoryBuilder.ts:127]: what is missing?

File: packages/cardano-services/environments/.env.sanchonet (hunk 1 above)
Comment (iccicci) [reply packages/cardano-services/environments/.env.sanchonet:4]: Actually the only available smash server is for mainnet, that's why we can't use for other networks

File: packages/cardano-services/src/ChainHistory/DbSyncChainHistory/ChainHistoryBuilder.ts (hunk 2 above)
Comment (iccicci) [reply packages/cardano-services/src/ChainHistory/DbSyncChainHistory/ChainHistoryBuilder.ts]: `governanceActionId`; e6131321d8 should solve this

**Review by mchappell — APPROVED** [review]

**Pull request merged**